.github/workflows/helm-version-update.yaml

```
@@ -0,0 +1,38 @@
+name: Build
+on:
+  schedule:
+    - cron: '0 8 * * 1'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  HELM_VERSION: v3.11.1
+
+jobs:
+  ensure-versions:
+    runs-on: ubuntu-22.04
+    permissions:
+      pull-request: write
```

```suggestion
  ensure-versions:
    runs-on: ubuntu-22.04
    
    permissions:
      content: write
      pull-request: write
```

Requires `content: write` to be able to commit the changes